### PR TITLE
[docs]: Fix documentation for the QR Scanner Component

### PR DESCRIPTION
**Pull request by void-hr**
**Description**
Updated qr-scanner docs as mentioned in issue #10928 

- [x] Content updates
- [x]  Formatting updates
- [x] Didn't changed any existing link 

This PR solves #10928 

Files on which changes are implemented -
Next - ToolJet/docs/docs/widgets/qr-scanner.md
Version 2.50.0 (LTS) - ToolJet/docs/versioned_docs/version-2.50.0-LTS/widgets/qr-scanner.md

Let me know if I am missing something else! Thanks for assigning me to this task 🚀

### CODE REVIEW

File: docs/docs/widgets/qr-scanner.md [hunk 1]
```diff
@@ -9,76 +9,78 @@ Scan QR codes using device camera and hold the data they carry.
...
+
+--- 
 
 ## Styles
 
 | <div style={{ width:"100px"}}> Style </div> | <div style={{ width:"100px"}}>  Description </div> | <div style={{ width:"100px"}}> Default Value </div> |
 |:------------ |:-------------|:--------- |
-| Visibility | Toggle on or off to control the visibility of the widget. You can programmatically change its value by clicking on the `Fx` button next to it. If `{{false}}` the widget will not be visible after the app is deployed. | By default, it's set to `{{true}}` |
-| Disable | This is `off` by default, toggle `on` the switch to lock the widget and make it non-functional. You can also programmatically set the value by clicking on the `Fx` button next to it. If set to `{{true}}`, the widget will be locked and becomes non-functional. | By default, its value is set to `{{false}}` |
+| Visibility | Toggle on or off to control the visibility of the component. You can programmatically change its value by clicking on the **fx** button next to it. If `{{false}}` the component will not be visible after the app is deployed. | By default, it's set to `{{true}}` |
```
Comment (RudraPratik30) [inline docs/docs/widgets/qr-scanner.md:83]: Hey @void-hr, can you please add a full stop here at the end of the sentence. Please do it for both the sentences of table and in both version of docs.

**Review by RudraPratik30 — CHANGES_REQUESTED** [review]

File: docs/docs/widgets/qr-scanner.md (hunk 1 above)
Comment (void-hr) [reply docs/docs/widgets/qr-scanner.md:83]: Yeah Sure!

**Comment by void-hr** [comment]
@RudraPratik30 changes reflected ! Can you take a look 👀

**Comment by RudraPratik30** [comment]
Hey @void-hr, thanks for contributing!
Approving the PR from my end and forwarding it to the team for final review.
Your PR will be merged by the end of next week. Thanks again!

**Review by RudraPratik30 — APPROVED** [review]

**Review by adishM98 — APPROVED** [review]

**Pull request merged**